package.json

```diff
@@ -121,6 +147,19 @@
               "path": "walkthrough/testing.png",
               "altText": "Testing"
             }
+          },
+          {
+            "id": "java.showTesting",
```

Comment:
This task is not shown possibly, because of its duplicated `id`. Is it on purpose?

Reply:
No, an oversight from copy/pasta.